src/Message/ExtendedResponse.php

```diff
@@ -0,0 +1,515 @@
+<?php
+
+
+namespace Omnipay\BlueSnap\Message;
+
+
+use DateTime;
+use DateTimeZone;
+use Omnipay\BlueSnap\Constants;
+use Omnipay\BlueSnap\CreditCard;
+use Omnipay\BlueSnap\Subscription;
+use Omnipay\BlueSnap\SubscriptionCharge;
+use Omnipay\BlueSnap\Transaction;
+use Omnipay\BlueSnap\Types;
+use Omnipay\Common\Message\RequestInterface;
+use SimpleXMLElement;
+
+/**
+ * BlueSnap response object for the Extended Payments API. See Request files or developer docs for usage.
+ *
+ * @link https://developers.bluesnap.com/v8976-Extended/docs
+ *
+ * @package Omnipay\BlueSnap\Message
+ */
+class ExtendedResponse extends AbstractResponse
```

Comment:
this entire file was created by extracting the extended API methods from `Response`